src/components/Articles.jsx

```diff
@@ -1,28 +1,33 @@
-import { useState, useEffect} from "react"
-import getArticles from "../../api-get"
+import { useState } from "react"
+import {useEffect} from "react"
+import {getArticles} from "../../api-get"
+import '../../src/Styling/article.css'
+
 
 export const Articles = () => {
     
-    const [articleList, setArticleList] = useState([])
-
-    useEffect (() => {
-        getArticles()
-        .then((body) =>{
-            console.log(body)
-            setArticleList(body)
-            console.log(body)
-        })
-        .catch((err) =>{
-            console.error(err)
-        })
-    }, [])
+const [articleList, setArticleList] = useState([])
 
-    return (
-        <div>
-            {articleList.map((article) =>{
-                <p>{article}</p>
-                console.log('article', article)
-            })}
-        </div>
-    )
+useEffect (() => {
+    getArticles()
```

Comment:
consider making getarticles return the articles directly rather than the the body which contains the articles, as the name suggests this is what it should return